lib/phone.rb

```diff
@@ -0,0 +1,22 @@
+require 'rubygems'
+require 'twilio-ruby'
+ 
+account_sid = "ACxxxxxxxxxxxxxxxxxxxxxxxx"
+auth_token = "yyyyyyyyyyyyyyyyyyyyyyyyy"
+client = Twilio::REST::Client.new account_sid, auth_token
+ 
+from = "+441263456072" # Your Twilio number
+ 
```

Comment:
MAgic numbers such as phone numbers should be stored as a constant at the top of the class.